src/test/kotlin/ecommerce/DBTest.kt

```diff
@@ -0,0 +1,98 @@
+package ecommerce
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest
+import org.springframework.jdbc.core.JdbcTemplate
+
+@JdbcTest
```

Comment:
I see this `@JdbcTest` annotation is a different choice from the previous
```kotlin
@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
```
which were in `CRUDTest`. What does `@JdbcTest` do?